### PR TITLE
Clear command and you language

### LINKED ISSUE #123: Update UG/DG with valid comments from other peer reviewers
Todo for UG
- [x] Add note to undo for macros
- [x] Add updated find to UG
- [x] Add updated clear to UG
- [x] Add updated Undo
- [x] Add updated GUI pictures.
- [x] Explicitly add hyperlink for appendix A
- [x] Update all command words with \`\`
- [x] Update terminology. Update MG with McGymmy
- [x] Define flag at the start of UG
- [x] Maybe update definition of Parameter
- [ ] Maybe shift warning into note (Danger box)
- [ ] <del>Maybe Add more hyperlinks to the command summary</del>
- [x] <b>_Maybe update_</b> You-language
- [ ] Maybe update phrasing for export command.
- [ ] Maybe crop/cut images

Todo for DG:
- [ ] **IGNORE** Introduce the name of the package more
- [x] Mention that MG and McGymmy are interchangeable
- [x] Update Glossary in DG
- [x] Add hyperlink to AddressBook3 in DG / ~~remove it~~
- [x] Define flag at the start of DG (In glossary)
- [ ] Standardise formatting
- [x] Remove data archiving
- [x] Fix Grammar / Spelling
- [ ] Maybe number the sections
- [ ] Maybe resize the images
- [x] **Maybe** Delete .puml tutorial note
- [x] Caps JSON

### PR DESCRIPTION
**Pull request by dcchan98**
Add Clear Command to UG
Update to more Youish language

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -41,18 +41,18 @@ Thus we recommend you to start by first understanding how McGymmy works at a hig
 
 ## Quick start
 
-Double click on the jar file to open the GUI.
-Type the command in the command box and press Enter to execute it. e.g. typing help and pressing Enter will open the help window.
+To get started, you will double click on the jar file to open the GUI.
```
Comment (Jh123x) [inline docs/UserGuide.md]: You will sound damn weird here
Maybe you can?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -83,13 +83,13 @@ For example in `add -n scrambled eggs -p 100`, `add` is a *Command Word* and `-n
 
 **How to intepret the each command's format**:
 
-We will follow the following convention for each command's format:
+You should follow the following convention for each command's format:
```
Comment (Jh123x) [inline docs/UserGuide.md]: I dont think this one need to change to you should?

**Review by Jh123x — CHANGES_REQUESTED** [review]
Mostly LGTM.

File: docs/UserGuide.md (hunk 1 above)
Comment (dcchan98) [reply docs/UserGuide.md]: Alright!

File: docs/UserGuide.md (hunk 2 above)
Comment (dcchan98) [reply docs/UserGuide.md]: Alright!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -364,18 +367,18 @@ Adds a macro to run several commands in succession.
...
 
-Format: `macro SHORTCUT ARGUMENT_1 ARGUMENT_2 ... ; COMMAND_1 ARGUMENTS_TO_COMMAND \REUSED_ARGUMENT; [COMMAND_2; ...]` <br>
+Format: `macro SHORTCUT PARAMETERS_1 PARAMETERS_2 ... ; COMMAND_1 PARAMETERS_TO_COMMAND \REUSED_PARAMETERS; [COMMAND_2; ...]` <br>
 
 <div markdown="block" class="alert alert-info">
 
 :information_source:
 
-* Create a macro with name `SHORTCUT` and arguments `ARGUMENT_1` and `ARGUMENT_2` which executes `COMMAND_1; COMMAND_2; ...`.
+* Create a macro with name `SHORTCUT` and parameters `PARAMETERS_1` and `PARAMETERS` which executes `COMMAND_1; COMMAND_2; ...`.
```
Comment (chewypiano) [inline docs/UserGuide.md]: I think can just remove these changes for macro, Joey made changes for this in a separate PR.

Comment (dcchan98) [reply docs/UserGuide.md]: Alright mate

**Review by Jh123x — APPROVED** [review]
LGTM

**Review by Jh123x — APPROVED** [review]
LGTM

**Pull request merged**